Interface/AddOns/nMainbar/config.lua

```diff
@@ -8,6 +8,7 @@ nMainbar.Config = {
         showVehicleKeybinds = true,
         showKeybinds = false,
         showMacronames = false,
+        OutOfRangeType = 'icon',     -- Changes how out of range coloring is displayed. Options: icon, hotkey.
```

Comment:
It is corrected just not in this commit. If you show all the changes it is in lowercase.